Gruntfile.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    uncss: {
+      dist: {
+        options: {
+          stylesheets: [
+            'assets/css/styles.css'
+          ]
+        }
+        files: {
+          'tidy.css': [
+            'http://127.0.0.1:4000/',
```

Comment:
Expected ']' to match '[' from line 14 and instead saw ','.<br>Missing semicolon.